src/apps/zharta/ethereum/zharta.lending-pool-core.contract-position-fetcher.ts

```diff
@@ -0,0 +1,101 @@
+import { Inject } from '@nestjs/common';
+import { BigNumberish, Contract } from 'ethers';
+
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { PositionTemplate } from '~app-toolkit/decorators/position-template.decorator';
+import { getLabelFromToken } from '~app-toolkit/helpers/presentation/image.present';
+import { DefaultDataProps } from '~position/display.interface';
+import { MetaType } from '~position/position.interface';
+import { isSupplied } from '~position/position.utils';
+import { ContractPositionTemplatePositionFetcher } from '~position/template/contract-position.template.position-fetcher';
+import {
+  GetDefinitionsParams,
+  DefaultContractPositionDefinition,
+  GetTokenDefinitionsParams,
+  UnderlyingTokenDefinition,
+  GetDisplayPropsParams,
+  GetTokenBalancesParams,
+} from '~position/template/contract-position.template.types';
+
+import { ZhartaContractFactory, ZhartaLendingPoolCore } from '../contracts';
+
+interface ZhartaLendingPoolCoreContractPositionDefinition extends DefaultContractPositionDefinition {
+  type: 'LENDING_POOL_CORE';
+}
+
+@PositionTemplate()
+export class EthereumZhartaLendingPoolCoreContractPositionFetcher extends ContractPositionTemplatePositionFetcher<Contract> {
```

Comment:
You want to extend the `Contract` you're calling, which in this case it `ZhartaLendingPoolCore`, same applies in the getContract, etc